.github/workflows/agent-refresh.yml

```
@@ -0,0 +1,541 @@
+            exit 0
+          fi
+          echo "Open promotion PR #${open_pr} is stale; rebuilding the candidate."
+          echo "skip=false" >> "$GITHUB_OUTPUT"
+
+      - name: Prepare immutable candidate reference
+        id: candidate
+        if: steps.skip.outputs.skip != 'true'
+        env:
+          UPSTREAM_COMMIT: ${{ steps.provenance.outputs.upstream_commit }}
+          UBUNTU_BASE_REFERENCE: ${{ steps.provenance.outputs.ubuntu_base_reference }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:3:1: upstream_short appears unused. Verify use (or export if used externally)

(shellcheck)